usr/src/test/zfs-tests/runfiles/smartos.run

```diff
@@ -31,9 +31,6 @@ tests = ['alloc_class_001_pos', 'alloc_class_002_neg', 'alloc_class_003_pos',
     'alloc_class_013_pos']
 tags = ['functional', 'alloc_class']
 
-[/opt/zfs-tests/tests/functional/cli_root/zpool_add]
```

Comment:
This file needs copyright update

Reply:
Because this was a mismerge, I didn't think it did.